src/styles/_temp-ds-tokens.css

```diff
@@ -0,0 +1,318 @@
+/**
```

Comment:
I know this file comes from somewhere else, but I think it might need changes? So I just want to make a comment so we hopefully don't forget to raise it as a question later.

- I believe the non-mono fonts need to be this:
  ```
  system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'
  ```
- I believe the mono font families need to be this:
  ```
  ui-monospace, Menlo, Consolas, Ubuntu Mono, monospace
  ```

Reply:
☝️Great call - this file comes from https://github.com/hashicorp/design-system-tokens/blob/main/dist/products/css/tokens.css . I think this is something we should flag with the design system team! 💯